apps/Client/tailwind.config.js

```diff
@@ -2,4 +2,14 @@
 export const content = [
   "./src/**/*.{js,ts,jsx,tsx}",
   "./public/index.html",
+  // Add other paths as needed
 ];
+export const theme = {
+  extend: {
+    colors: {
+      orange: {
+        500: '#ff9900', // Adjust the hex color code as needed
```

Comment:
Why the comment? You don't need to adjust the hex color. Find the one suitable and fix there, and remove the comment. The comment is not needed.